packages/apollo-server-core/src/plugin/schemaReporting/index.ts

```diff
@@ -98,21 +99,19 @@ export function ApolloServerPluginSchemaReporting(
         }
       }
 
-      if (schemaIsFederated(schema)) {
-        throw Error(
-          [
-            'Schema reporting is not yet compatible with federated services.',
-            "If you're interested in using schema reporting with federated",
-            'services, please contact Apollo support. To set up managed federation, see',
-            'https://go.apollo.dev/s/managed-federation',
-          ].join(' '),
-        );
-      }
-
-      const executableSchema = overrideReportedSchema ?? printSchema(schema);
-      const executableSchemaId = computeExecutableSchemaId(executableSchema);
-
+      const isFederatedSchemaReporting = schemaIsFederated(schema);
       if (overrideReportedSchema !== undefined) {
+        if (isFederatedSchemaReporting) {
```

Comment:
I'm pretty sure schemaIsFederated is looking for the case that this is a *subgraph* schema, so neither of the places you use this bool seem right.

Are we planning to do anything about schema reporting from subgraphs?

Reply:
You're right, good catch! I've reverted this part of the code, so that we use `schemaIsFederated` only to throw for schemas that look like implementing service schemas.

>Are we planning to do anything about schema reporting from subgraphs?

I'm not in the loop enough about future plans there. For now I think we intend to keep the status quo, but we might start having reporting there in the future.